lua/entities/gmod_wire_pod.lua

```diff
@@ -99,12 +99,16 @@ end
 
 -- Server
 
+local pods = {}
```

Comment:
`local pods = pods or {}`

Reply:
pods would always be nil unless there was a global defined, so this wouldn't work